### ISSUE TITLE
Strange NPE

### ISSUE THREAD

**Issue by Stealth2800** [FlexSeries/FlexChat#9]
Strange NPE that occurs after installed on a Towny server that has been running for almost a day.  Seems to be related to the TownyTownChannelInstance's Town variable being null for some reason. Haven't been able to replicate issue.

```
[19:53:55] [Thread-3576/INFO]: UUID of player - is -
[19:53:55] [Server thread/WARN]: java.lang.NullPointerException
[19:53:55] [Server thread/WARN]:        at me.st28.flexseries.flexchat.hooks.towny.TownyTownChannelInstance.getDisplayName(TownyTownChannelInstance.java:49)
[19:53:55] [Server thread/WARN]:        at me.st28.flexseries.flexchat.api.channel.Channel.getInstance(Channel.java:162)
[19:53:55] [Server thread/WARN]:        at me.st28.flexseries.flexchat.api.chatter.Chatter.load(Chatter.java:71)
[19:53:55] [Server thread/WARN]:        at me.st28.flexseries.flexchat.backend.chatter.ChatterManagerImpl.loadPlayerChatter(ChatterManagerImpl.java:116)
[19:53:55] [Server thread/WARN]:        at me.st28.flexseries.flexchat.backend.chatter.ChatterManagerImpl.loadPlayer(ChatterManagerImpl.java:139)
[19:53:55] [Server thread/WARN]:        at me.st28.flexseries.flexlib.player.data.PlayerLoader.loadProvider(PlayerLoader.java:141)
[19:53:55] [Server thread/WARN]:        at me.st28.flexseries.flexlib.player.data.PlayerLoader.load(PlayerLoader.java:90)
[19:53:55] [Server thread/WARN]:        at me.st28.flexseries.flexlib.player.PlayerManager.loadPlayer(PlayerManager.java:243)
[19:53:55] [Server thread/WARN]:        at me.st28.flexseries.flexlib.player.PlayerManager.onPlayerJoin(PlayerManager.java:280)
[19:53:55] [Server thread/WARN]:        at sun.reflect.GeneratedMethodAccessor248.invoke(Unknown Source)
[19:53:55] [Server thread/WARN]:        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[19:53:55] [Server thread/WARN]:        at java.lang.reflect.Method.invoke(Method.java:497)
[19:53:55] [Server thread/WARN]:        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306)
[19:53:55] [Server thread/WARN]:        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[19:53:55] [Server thread/WARN]:        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502)
[19:53:55] [Server thread/WARN]:        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.PlayerList.onPlayerJoin(PlayerList.java:298)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.PlayerList.a(PlayerList.java:157)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.LoginListener.b(LoginListener.java:144)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.LoginListener.c(LoginListener.java:54)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.NetworkManager.a(NetworkManager.java:231)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.ServerConnection.c(ServerConnection.java:148)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:872)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:378)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:711)
[19:53:55] [Server thread/WARN]:        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:614)
[19:53:55] [Server thread/WARN]:        at java.lang.Thread.run(Thread.java:745)
[19:53:55] [Server thread/ERROR]: [FlexLib/players] Provider 'me.st28.flexseries.flexchat.backend.chatter.ChatterManagerImpl' failed to load data for player '-' (-): null
```

**Comment by Stealth2800** [comment]
A new theory is that there is a discrepancy between Towny's player and town data.  The player data thinks it's in a town, but the town doesn't actually exist.

**Comment by Stealth2800** [comment]
Appears to be fixed by 0a51fdd02f4f9ce6fba9597c9e2612002c022410
